package.json

```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "@emotion/core": "^10.0.17",
     "@emotion/styled": "^10.0.17",
-    "@types/jest": "^24.0.18",
+    "@types/jest": "^24.0.23",
```

Comment:
q: Shouldn't `Jest` be a `devDependency`

Reply:
Yep, I have a whole bunch of deps that should be dev dep's.

Raised this as an issue.